docs/basic-usage.md

```diff
@@ -43,6 +43,13 @@ If you ever need to completely remove all settings from their persistent storage
 service('settings')->flush();
 ```
 
+Also, you can use the `pull()` method to retrieve a setting and then remove it from the persistent storage in one go. This is useful when you want to retrieve a value and ensure it is no longer available for future use.
+
+```php
+// The same as config('App')->siteName;
+$siteName = service('settings')->pull('App.siteName');
```

Comment:
I don't think this comment is accurate. Like... I know what you mean, but it could be misinterpreted as if using `config(‘App’)->siteName` and `service(‘settings’)->pull(‘App.siteName’)` is the same thing, which it is not.